.github/actions/tox-run-action/action.yaml

```diff
@@ -0,0 +1,64 @@
+---
+name: "tox-run"
+description: "Run tox on specific environments"
+
+inputs:
+  tox-dir:
+    description: "Directory containing tox.ini file"
+    required: false
+    default: "."
+  py-version:
+    decription: "Version of python to use"
+    required: false
+    default: "3.12"
+  tox-envs:
+    description: "Tox envs to run on this py-version"
+    required: true
+  parallel:
+    description: "Whether to run jobs in parallel"
+    required: false
+    default: "true"
+
+runs:
+  using: "composite"
+  steps:
+    - id: setup-python
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ inputs.py-version }}
+    - id: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install tox
+        pip freeze
+    - id: Run tox
+      # yamllint disable rule:line-length
+      run: |
+        cd "${GITHUB_WORKSPACE}/${{ inputs.tox-dir }}" || exit 1
+
+        if [[ -f "requirements.txt" ]]; then
+            pip install -r requirements.txt
+        fi
+        if [[ -f "requirements-test.txt" ]]; then
+            pip install -r requirements.txt
```

Comment:
You're correct. However, per @tykeal's comments received via Slack, I think I'm going to yank a fair amount of the prep/installing stuff, so I think this will get removed entirely.